### DISCUSSION TITLE
🤔 [QUESTION] Mix图表如何统一多个图表的y轴？

### DISCUSSION THREAD

**Discussion by NecisDJ** [antvis/G2Plot#3581] [Q&A] [answered]
### 🐛 Question description
Mix图表如何统一多个图表的y轴？
https://antv-g2plot.gitee.io/zh/examples/plugin/multi-view#combo-plot
按照demo上的写法，似乎是配置写死了y轴的最大值？在面对动态数据的时候，是否有别的解决方案？

**Comment by hustcc** [comment, marked as answer]
参考：https://github.com/antvis/G2Plot/discussions/3565
